notebooks/COS/ExceptionReport/ExceptionReport.ipynb

```diff
@@ -0,0 +1,1448 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    
</pre><p> </p><p>Please follow the suggestion given in the example above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>